doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -198,6 +198,35 @@ system.
 | `id`                | [`text`]         | The ID of the source. Corresponds to [`mz_catalog.mz_sources.id`](../mz_catalog#mz_sources).                   |
 | `replication_slot`  | [`text`]         | The name of the replication slot in the PostgreSQL database that Materialize will create and stream data from. |
 
+<!-- ### `mz_prepared_statement_history` -->
+
+<!-- The `mz_prepared_statement_history` table contains a subset of all -->
+<!-- statements that have been prepared. It only contains statements that -->
+<!-- have one or more corresponding executions in inclusion in -->
+<!-- [`mz_statement_execution_history`](#mz_statement_execution_history). -->
+
+<!-- | Field         | Type                         | Meaning                                                                                                                           | -->
+<!-- |---------------|------------------------------|-----------------------------------------------------------------------------------------------------------------------------------| -->
+<!-- | `id`          | [`uuid`]                     | The globally ID of the prepared statement.                                                                                        | -->
+<!-- | `session_id`  | [`uuid`]                     | The globally unique ID of the session that prepared the statement. Corresponds to [`mz_session_history.id`](#mz_session_history). | -->
+<!-- | `name`        | [`text`]                     | The name of the prepared statement (the default prepared statement's name is the empty string).                                   | -->
+<!-- | `sql`         | [`text`]                     | The SQL text of the prepared statement.                                                                                           | -->
+<!-- | `prepared_at` | [`timestamp with time zone`] | The time at which the statement was prepared.                                                                                     | -->
```

Comment:
I'm not sure this has any value, because we don't even resolve object names on the stored statement, so users could prepare something that references an object that doesn't yet exist.

Reply:
You're saying you don't think the `prepared_at` timestamp has value? Maybe not, but I'd rather keep it in case we decide we want it, as it's only a few additional bytes.